cirq-google/cirq_google/serialization/circuit_serializer_test.py

```diff
@@ -226,6 +226,15 @@ def circuit_proto(json: Dict, qubits: List[str]):
             }
         ),
     ),
+    (
+        cirq.FSimGate(theta=0.5, phi=0.0)(Q0, Q1),
+        op_proto(
+            {
+                'fsimgate': {'theta': {'float_value': 0.5}, 'phi': {'float_value': 0.0}},
+                'qubit_constant_index': [0, 1],
+            }
+        ),
+    ),
```

Comment:
Maybe also add some cases that explicitly use int for theta and phi?

Reply:
Added.  Also added a case for a sympy formula.